learning/tour-of-beam/frontend/lib/config.dart

```diff
@@ -18,22 +18,17 @@
 
 // TODO(alexeyinkin): Generate this file on deployment.
 
-const _cloudFunctionsProjectRegion = 'us-central1';
-const _cloudFunctionsProjectId = 'tour-of-beam-2';
+const _cloudFunctionsProjectRegion = 'us-east1';
+const _cloudFunctionsProjectId = 'tob-demo-env1';
 const cloudFunctionsBaseUrl = 'https://'
     '$_cloudFunctionsProjectRegion-$_cloudFunctionsProjectId'
     '.cloudfunctions.net';
 
 // Copied from Playground's config.g.dart
 
 const String kAnalyticsUA = 'UA-73650088-2';
-const String kApiClientURL =
-    'https://backend-router-beta-dot-apache-beam-testing.appspot.com';
-const String kApiJavaClientURL =
-    'https://backend-java-beta-dot-apache-beam-testing.appspot.com';
-const String kApiGoClientURL =
-    'https://backend-go-beta-dot-apache-beam-testing.appspot.com';
-const String kApiPythonClientURL =
-    'https://backend-python-beta-dot-apache-beam-testing.appspot.com';
-const String kApiScioClientURL =
-    'https://backend-scio-beta-dot-apache-beam-testing.appspot.com';
+const String kApiClientURL = 'https://router.34.120.66.174.nip.io/';
+const String kApiJavaClientURL = 'https://java.34.120.66.174.nip.io/';
+const String kApiGoClientURL = 'https://go.34.120.66.174.nip.io/';
+const String kApiPythonClientURL = 'https://python.34.120.66.174.nip.io/';
+const String kApiScioClientURL = 'https://scio.34.120.66.174.nip.io/';
```

Comment:
Please only keep the production URLs in the branch. If someone needs to set up a stage with other URLs, they should replace the URLs after checkout.

Reply:
Still non-production URLs are there.